yaksh/models.py

```diff
@@ -1182,7 +1199,17 @@ def make_answerpaper(self, user, ip, attempt_num, course_id):
             ans_paper.save()
             questions = self._get_questions_for_answerpaper()
             ans_paper.questions.add(*questions)
-            question_ids = [str(que.id) for que in questions]
+            question_ids = []
+            for question in questions:
+                question_ids.append(str(question.id))
+                testcases = question.get_test_cases()
+                if question.shuffle_testcases:
+                    random.shuffle(testcases)
+                testcases_ids = ",".join([str(tc.id) for tc in testcases])
```

Comment:
It is not clear why we need the order for every question even though shuffling makes no sense for assert, stdio, fill in the blanks etc.